assets/sass/components/_badges-tags.scss

```diff
@@ -1,11 +1,11 @@
 .govuk-table {
   a.govuk-link {
-    font-weight: normal;
+    font-weight: normal ;
```

Comment:
extra space here before the `;` - my previous comment about adding a space was about adding a space between the property (`font-weight:`) and the value: (`normal`), apologies if that wasn't clear. It looks like you've got this in the other files too. Would you mind updating those too?

So just as an example, you'd have:
`white-space: nowrap;`